frontend/zc_messaging/src/routes/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
```

Comment:
this idea of having a route folder is kinda strange to me, haven't seen this in react projects. Is there a reason 🤔

Reply:
The route folder holds components relating to routing. Currently, there are two components for routing.

Reply:
okay